app/views/coronavirus/pages/_form.html.erb

```diff
@@ -0,0 +1,77 @@
+<%= render "govuk_publishing_components/components/input", {
+  label: {
+    text: "Title",
+    bold: true,
+  },
+  name: "landing_page[header_title]",
+  value: @page.header_title,
+  id: "header_title",
+  error_message: error_items(@page.errors.messages, :header_title),
+} %>
+
+<%= render "components/markdown_editor", {
+  label: {
+    text: "Header body",
+    bold: true
+  },
+  textarea: {
+    name: "landing_page[header_body]",
+    id: "header_body",
+    rows: 20,
+    value: @page.header_body
+  },
+  error_message: error_items(@page.errors.messages, :header_body),
+  controls: [:headings, :numbered_list, :bullets]
+} %>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Header action link",
+  heading_level: 4,
+  margin_bottom: 5
+} %>
+
+<%= render "govuk_publishing_components/components/hint", {
+  text: "Add content across two lines for the header action link. This allows mobile users to see the link text correctly wrap over 2 lines in a block, rather than allowing their device to choose."
+} %>
+
+<%= render "govuk_publishing_components/components/input", {
+  label: {
+    text: "Action link text line one",
+    bold: true,
+  },
+  name: "landing_page[header_link_pre_wrap_text]",
+  value: @page.header_link_pre_wrap_text,
+  id: "header_link_pre_wrap_text",
+  hint: "40 characters allowed. Any content which goes over the limit should be added as part of line 2.",
+  error_message: error_items(@page.errors.messages, :header_link_pre_wrap_text),
+} %>
```

Comment:
Might want to consider adding some visual feedback on the character length by using the [character count component](https://components.publishing.service.gov.uk/component-guide/character_count). By not waiting for the form to be submitted before displaying an error when the character count is > 40, I feel it would be a much nicer user experience.